pkg/kando/location_pull.go

```diff
@@ -59,7 +61,14 @@ func runLocationPull(cmd *cobra.Command, args []string) error {
 		return err
 	}
 	s := pathFlag(cmd)
+	id := backupIDFlag(cmd)
 	ctx := context.Background()
+	if p.Location.Type == crv1alpha1.LocationTypeKopia {
```

Comment:
Can we merge the profile changes with this PR and open a separate one for these changes?

Reply:
Sounds good!